GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -836,6 +857,42 @@ private async Task ExecuteAutoOnboardAsync(ExecutionContext executionContext, IL
             }
         }
 
+        [Disable("ALL_FUNCTIONS_DISABLED")]
+        [FunctionName("Heartbeat")]
+        public void Heartbeat([TimerTrigger("0 * * * * *", RunOnStartup = false)] TimerInfo timerInfo, ExecutionContext executionContext, ILogger logger)
+        {
+            using Activity SessionActivity = GetInvocationActivity(executionContext, timerInfo);
+            // Todo : add monitoring info to heartbeat (poison queue checks, failed session invocation IDs)
+            OpenTelemetryMetric.HeartbeatCounter.Add(1);
+        }
+
+        private Activity GetInvocationActivity(ExecutionContext context, string queueItem, int dequeueCount)
```

Comment:
To me naming these `StartActivity` would imply that this method is returning an Activity which is already started. what do you think about `ConstructInvocationActivty()`

Reply:
But that's exactly what you're doing?

Reply:
Yes you are certainly right. This was not the behavior I intended. I have move the start calls to the function definitions which is better because it will give a better indication of where this activity is being used in traces.